<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkeletonModifyClass</class>
 <widget class="QMainWindow" name="SkeletonModifyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1669</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkeletonModify</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>1440</width>
      <height>540</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>140</y>
      <width>181</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>PointsSelect</string>
     </attribute>
     <widget class="QCheckBox" name="checkboxPoint16">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>16</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint13">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>13</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint10">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint17">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>17</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint15">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>15</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>91</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint19">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>19</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint12">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint18">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>18</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint14">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>14</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint20">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint11">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>11</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkboxPoint0">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonPreFrame">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Pre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNextFrame">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label1_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string> CurrFrame:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCurrFrame">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: rgb(255, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label1">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>TotalFrame:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTotalFrame">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: rgb(255, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveToFile">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>570</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Save Change To File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1669</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>    File    </string>
    </property>
    <addaction name="actionLoadConfigureFile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionLoadConfigureFile">
   <property name="text">
    <string>Load Configure File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SkeletonModify.qrc"/>
 </resources>
 <connections/>
</ui>
